public/v4/apps/maybe-finance.yml

```diff
@@ -0,0 +1,69 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: ghcr.io/maybe-finance/maybe:$$cap_app_version
+        volumes:
+            - $$cap_appname-storage:/rails/storage
+        restart: unless-stopped
+        environment:
+            SELF_HOSTED: 'true'
+            RAILS_FORCE_SSL: 'false'
+            RAILS_ASSUME_SSL: 'false'
+            GOOD_JOB_EXECUTION_MODE: 'async'
+            SECRET_KEY_BASE: $$cap_pass
+            SYNTH_API_KEY: $$cap_api_key
+            DATABASE_URL: postgresql://maybe_user:$$cap_pg_pass@srv-captain--$$cap_appname-db:5432/maybe_production
+        caproverExtra:
+            containerHttpPort: '3000'
+        depends_on:
+            $$cap_appname-db:
+                condition: service_healthy
+    $$cap_appname-db:
+        image: postgres:16
+        restart: always
+        volumes:
+            - $$cap_appname-postgres-data:/var/lib/postgresql/data
+        environment:
+            POSTGRES_USER: maybe_user
+            POSTGRES_DB: maybe_production
+            POSTGRES_PASSWORD: $$cap_pg_pass
+        healthcheck:
```

Comment:
healthcheck isn't currently being consumed by CapRover. Let's remove to avoid confusion.